wallet/core/src/account/mod.rs

```diff
@@ -348,6 +354,57 @@ pub trait Account: AnySync + Send + Sync + 'static {
         Ok((generator.summary(), ids))
     }
 
+    async fn pskb_from_send_generator(
+        self: Arc<Self>,
+        destination: PaymentDestination,
+        priority_fee_sompi: Fees,
```

Comment:
just wondering how priority fee will be distibuten between txs? is it accumulative or per tx fee?

Reply:
Per tx fee.
I've added a verifier function to loop through UTXOs and check if fee is coverable for each one.

Reply:
Could you add doc comment describing it

Reply:
Added and function named changed to an explicit one.